tests/agent_server/test_server_agent.py

```diff
@@ -2137,6 +2137,7 @@ async def test_reload(
     assert dep_state.index == resource_container.Provider.reloadcount("agent1", "key2")
 
 
+@pytest.mark.skip("Skipped for agent refactor")
```

Comment:
We have to be more descriptive in this message. Why do we need to skip this test?